assets/src/domain/eventEditor/ui/datetimes/datesList/List/useList.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Datetime } from '../../../../services/apollo/types';
+
+import useEntityListFilterState from '@appLayout/entityList/filterBar/useEntityListFilterState';
+import useDatesListFilterState from '../filterBar/useDatesListFilterState';
+import { useEntityPagination } from '@appLayout/entityList/pagination';
+
+const useList = (datetimes: Datetime[]): any => {
```

Comment:
IMHO, this should be named as `useFilteredDatetimes`

Reply:
yes, that was quick name/ file to abstract logic from UI